src/flamenco/runtime/context/fd_exec_txn_ctx.c

```diff
@@ -16,7 +16,7 @@ fd_exec_txn_ctx_new( void * mem ) {
     return NULL;
   }
 
-  // fd_memset(mem, 0, FD_EXEC_TXN_CTX_FOOTPRINT);
+  fd_memset(mem, 0, FD_EXEC_TXN_CTX_FOOTPRINT);
```

Comment:
Would be good to get context from @ripatel-fd on why this was commented out - was it for performance?

Reply:
You're right - it was commented out for performance reasons. I've restored it and instead opted to manually set txn_ctx fields to default values within `_context_create`